e2e-tests/playwright/e2e/guest-signin-happy-path.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { HomePage } from '../support/pages/HomePage';
+import { Common } from '../utils/Common';
+import { templates } from '../support/testData/templates';
+
+test.describe('Guest Signing Happy path', () => {
+  let uiHelper: UIhelper;
+  let homePage: HomePage;
+  let common: Common;
+
+  test.beforeEach(async ({ page }) => {
```

Comment:
@subhashkhileri  With beforeAll the tests are failing, looking into some articles it's preferable to use beforeEach with Playwright (it's different if compared with Cypress) due some reasons, like:
- Clean State: beforeEach ensures that each test starts with a clean slate. This is crucial for test reliability because tests should not depend on the side effects of other tests. If one test modifies the application state, beforeEach can reset these changes, so subsequent tests aren't affected.
- No Side Effects: Tests should ideally be independent and capable of being run in any order. Using beforeEach helps maintain this by setting up the test environment freshly for each test, reducing the risk of tests affecting each other.

Reply:
@gustavolira In Cypress, we force it to retain the state to avoid logging in at every step/action. In an ideal scenario, an end user wouldn't perform login and logout for each action, so we need to run it continuously. Regarding failures due to beforeAll, I have implemented a similar approach in one of the test cases; perhaps we can handle it in the same way.